src/ui/hooks/useToasts.ts

```diff
@@ -17,17 +23,28 @@ export default function useToasts(): UseToasts {
   const [toasts, setToasts] = useState<Toast[]>([]);
 
   const addToast = useCallback(
-    (message: string, variant: "success" | "danger") => {
+    (message: string, variant: "success" | "danger", permanent?: boolean) => {
```

Comment:
Having boolean args often leads to confusion about what they actually do. Firefox has a pattern where you do `/* argName = */ true` for boolean args. Here, I think we should use an object, e.g.

```
type AddToastParams = {
  message: string;
  variant: "success" | "danger";
  permanent: boolean;
};
const addToast = useCallback({ message, variant, permanent = false } = {}: AddToastParams) => { /* ... */ });
```

Also, I'd rename the "permanent" parameter to "autohide"